--- sbcl-0.9.0/src/runtime/x86-arch.c.~1.27.~	2004-09-09 14:10:15.000000000 +0200
+++ sbcl-0.9.0/src/runtime/x86-arch.c	2005-05-02 13:57:58.195396439 +0200
@@ -85,7 +85,7 @@
 	    vlen = *(char*)(*os_context_pc_addr(context))++;
 	    /* Skip Lisp error arg data bytes. */
 	    while (vlen-- > 0) {
-		( (char*)(*os_context_pc_addr(context)) )++;
+		( *(char**)(os_context_pc_addr(context)) )++;
 	    }
 	    break;
 
@@ -265,12 +265,12 @@
 	break;
 
     case trap_Breakpoint:
-	(char*)(*os_context_pc_addr(context)) -= 1;
+	*(char**)(os_context_pc_addr(context)) -= 1;
 	handle_breakpoint(signal, info, context);
 	break;
 
     case trap_FunEndBreakpoint:
-	(char*)(*os_context_pc_addr(context)) -= 1;
+	*(char**)(os_context_pc_addr(context)) -= 1;
 	*os_context_pc_addr(context) =
 	    (int)handle_fun_end_breakpoint(signal, info, context);
 	break;
